src/main/webapp/app/pages/ActionableGenesPage.tsx

```diff
@@ -540,6 +558,102 @@ export default class ActionableGenesPage extends React.Component<
   ];
 
   render() {
+    const levelSelectionSection = [];
+    for (const key in LEVEL_TYPES) {
+      if (LEVEL_TYPES[key]) {
+        levelSelectionSection.push(
+          <>
+            <Row
```

Comment:
Can we move the row to a functional component? It would be easier to look at the logics.

Reply:
I created a functional component, but I met a issue that I'm confused with. Please see the below comment.